gcloud-java-core/src/main/java/com/google/gcloud/BaseServiceException.java

```diff
@@ -62,17 +68,25 @@ public Integer code() {
     }
 
     /**
+     * Returns true if the error indicates that the API call was certainly not accepted by the
+     * server.
```

Comment:
Maybe an example could help users understand what `rejected()` means? We could mention that `rejected()` can be `true` for rate limit exceeded exceptions.